WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -46,11 +48,12 @@ import java.util.SortedMap
 import javax.inject.Inject
 import javax.inject.Named
 import kotlin.coroutines.experimental.Continuation
-import kotlin.coroutines.experimental.suspendCoroutine
 
 private const val ACTION_DELAY = 100
 private const val SEARCH_DELAY = 200
+private const val SNACKBAR_DELAY = 500
 private const val SEARCH_COLLAPSE_DELAY = 500
+private const val PAGE_UPLOAD_TIMEOUT = 5000L
```

Comment:
Is this timeout based on another constant or arbitrary?

Reply:
Short answer - gut feeling 😆